root/static/scripts/tests/relationship-editor.js

```diff
@@ -700,6 +700,8 @@ relationshipEditorTest("hidden input fields are generated for non-release forms"
         "edit-artist.rel.2.period.end_date.year" : "",
         "edit-artist.rel.2.period.ended": "1",
         "edit-artist.rel.2.backward": "1",
+        "edit-artist.rel.2.entity0_credit": "",
+        "edit-artist.rel.2.entity1_credit": "",
```

Comment:
These now exist because these tests are for new relationships, where every field is considered "changed" to start with.